danger-klaxit/lib/danger_plugin.rb

```diff
@@ -56,12 +56,9 @@ def fail_for_bad_commits
 
   # Start a scan using `danger-brakeman_scanner`. Arguments are deduced from
   # the current path, and klaxit naming conventions (i.e `klaxit-<name>`).
-  def run_brakeman_scanner(configs = { app_path: "." })
+  def run_brakeman_scanner
+    configs = { app_path: "." }
```

Comment:
Je pense qu'on pourrait garder ce changement, ça permet de paramétrer plus facilement brakeman.

Reply:
Pas d'accord, mais j'ai fixé quand même.
Si demain on passe à `run_brakeman_scanner` un objet de config, alors, et si on fait pas gaffe, on pourrait oublier de passer la clef `app_path`. Ca pètera les tests et donc on fixera rapidement, mais en omettant le paramètre alors même qu'il est pas utilisé, on s'assure que quand on voudra le lui passer, on implémentera correctement l'ajout des param par défaut.
Au pire,  un : 
```ruby
def run_brakeman_scanner(configs = {  })
   configs[:app_path] ||=  "." 
```

Reply:
@JoakimKlaxit I think your idea is better than mine! I'll let you choose, both are OK IMHO.